image create now accepts a storage-sku.

adewaleo opened the pull request:
image create now accepts a storage-sku. Unused if source vm specified, due to service side issue. 

Also updated help text and changed some model imports to use get_models

closes #7115 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

yugangw-msft commented on src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py:
```diff
@@ -40,10 +40,9 @@ def load_arguments(self, _):
+
+    if StorageAccountTypes:
+        disk_sku = CLIArgumentType(arg_type=get_enum_type(StorageAccountTypes))
     else:
```
let us get rid of this `else`

adewaleo replied:
@yugangw-msft, i added this because .get_models(storageaccounttypes) fails for some  profiles.

See this [job](https://travis-ci.org/Azure/azure-cli/jobs/443465561#L1341) in this previous CI [build](https://travis-ci.org/Azure/azure-cli/builds/443465552). Also see the other failed job in the build.

adewaleo replied:
I will open an issue in our repo and reference the service end bug.

adewaleo commented:
@tjprescott, I added some test coverage for the parameter I added to image create and opened the issue to track the service end bug. Also @yugangw-msft  and I got to the bottom of the profile issue we were discussing before scrum.

If CI passes, this should be good to merge.

kyuz0 commented:
A question: as this is used to create images, how are people supposed to take advantage of --storage-sku if it's ignored when specifying a source VM: I mean, isn't that a mandatory parameter to create an image? How would one create an image without a source VM?
